visidata/loaders/_pandas.py

```diff
@@ -67,7 +70,9 @@ def reload(self):
             self.df = readfunc(str(self.source), **options('pandas_'+filetype+'_'))
 
         # reset the index here
-        if type(self.df.index) is not pd.RangeIndex:
+        if type(self.df.index) is pd.Int64Index:
```

Comment:
Really nice find!

Should we keep an eye out for other Indexing types?

Reply:
Not really sure to be honest. Converting Int64-->Range here seems clear enough here (assuming we want/need a RangeIndex), but I don't know enough to extrapolate beyond that.

Reply:
Okay, perhaps this is the best we can do for now, and we can mature it from there. I did test a few formats, and things seem to be fine. =) Perhaps folks who care about other formats can chip in if they notice something. Thanks!